google-cloud-bigtable/src/test/java/com/google/cloud/bigtable/data/v2/stub/metrics/CompositeTracerTest.java

```diff
@@ -149,11 +149,11 @@ public void testAttemptCancelled() {
   public void testAttemptFailed() {
     RuntimeException error = new RuntimeException();
     Duration delay = Duration.ofMillis(10);
-    compositeTracer.attemptFailed(error, delay);
```

Comment:
same here, I think we should keep calling attemptFailed and verify attemptFailedDuration gets called?